src/web_app_skeleton/src/actions/errors/show.cr.ecr

```diff
@@ -80,7 +80,7 @@ class Errors::Show < Lucky::ErrorAction
 
   private def error_html(message : String, status : Int)
     context.response.status_code = status
```

Comment:
This line could be removed, right?

Reply:
No, it's used on the exception page https://github.com/crystal-loot/exception_page/blob/aa641f3905b4f4be68c8f569d16f793fc1338873/src/exception_page.cr#L75 :unamused: